Source/Core/Core/HW/DSPHLE/UCodes/AXStructs.h

```diff
@@ -7,87 +7,45 @@
 
 namespace DSP::HLE
 {
+struct VolumeData
+{
+  u16 volume;
+  u16 volume_delta;
+};
+
 struct PBMixer
 {
-  u16 left;
-  u16 left_delta;
-  u16 right;
-  u16 right_delta;
-
-  u16 auxA_left;
-  u16 auxA_left_delta;
-  u16 auxA_right;
-  u16 auxA_right_delta;
-
-  u16 auxB_left;
-  u16 auxB_left_delta;
-  u16 auxB_right;
-  u16 auxB_right_delta;
-
-  u16 auxB_surround;
-  u16 auxB_surround_delta;
-  u16 surround;
-  u16 surround_delta;
-  u16 auxA_surround;
-  u16 auxA_surround_delta;
+  VolumeData main_left, main_right;
```

Comment:
This might just be my personal preference here, but I'd rather have this on two lines.
Mostly inconsequential here, since there are no type modifiers (like `*`, `const` or whatever); but for consistency I feel it makes more sense (especially with the `surround` ones being ordered differently).
```suggestion
  VolumeData main_left;
  VolumeData main_right;
```

Thoughts?

Reply:
I kinda liked "left" being on the left and "right" being on the right^^